test/e2e/kcm_operator.go

```
@@ -0,0 +1,223 @@
+
+	output, err := oc.AsAdmin().WithoutNamespace().Run("whoami").Args("--show-server").Output()
+	o.Expect(err).NotTo(o.HaveOccurred())
+	routeS := strings.Split(strings.Split(output, "api")[1], ":")[0]
+	internalLB := "server: https://api-int" + routeS
+
+	output, err = oc.AsAdmin().WithoutNamespace().Run("get").Args("configmap", "controller-manager-kubeconfig", "-n", operatorclient.TargetNamespace, "-oyaml").Output()
+	o.Expect(err).NotTo(o.HaveOccurred())
+	o.Expect(output).To(o.ContainSubstring(internalLB))
+
+	leaderKcm := testlib.GetLeaderKCM(oc)
+	defer testlib.DebugNodeWithChroot(oc, leaderKcm, "mv", "/home/kube-apiserver-pod.yaml", "/etc/kubernetes/manifests/")
```

golangci-lint (2.12.2)

[error] Error return value of `testlib.DebugNodeWithChroot` is not checked

(errcheck)